src/filefinder.cpp

```diff
@@ -447,9 +492,26 @@ std::string FileFinder::FindFont(const std::string& name) {
 static void add_rtp_path(const std::string& p) {
 	using namespace FileFinder;
 	std::shared_ptr<DirectoryTree> tree(CreateDirectoryTree(p));
-	if(tree) {
+	if (tree) {
 		Output::Debug("Adding %s to RTP path", p.c_str());
-		search_paths.push_back(tree);
+		rtp_state.search_paths.push_back(tree);
+
+		auto hit_info = RTP::Detect(tree, atoi(Player::GetEngineVersion().c_str()));
```

Comment:
`Player::EngineVersion` ?

Reply:
hm, yes. That's exactly the function I needed, thx :D